mailbag/derivatives/eml.py

```diff
@@ -0,0 +1,62 @@
+# This is an example derivative, meant to show how
+# to hook up a real parser
+
+# Does nothing currently
+from os.path import join
+import mailbag.helper as helper
+import os,glob
+import mailbox
+from mailbag.email_account import EmailAccount
+from structlog import get_logger
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email import generator
+from mailbag.derivative import Derivative
+#from mailbag.controller import Controller
+
+log = get_logger()
+class ExampleDerivative(Derivative):
+    derivative_name = 'eml'
+
+    def __init__(self,email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        a = message.Message
+        print("Message",args.directory)
+        print(args.mailbag_name)
+
+
+
+        msg = MIMEMultipart('alternative')
+
+        msg['Subject'] = message.Subject
+        msg['From'] = message.From
+        msg['To'] = message.To
+        msg['Cc'] = message.Cc
+        msg['Bcc'] = message.Bcc
+
+
+        if(message.Message is not None):
+            msg.attach(message.Message)
+        elif (message.Headers is not None):
```

Comment:
In the case where `message.Message` is none, `message.headers` should also be present and is also a message object. You would just have to add `message.HTML_Body` and `message.Test_Body` to it.